presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/thrift/StaticMetastoreConfig.java

```diff
@@ -31,6 +31,7 @@
 
     private List<URI> metastoreUris;
     private String metastoreUsername;
+    private boolean isMultipleMetastoreEnabled;
```

Comment:
Let's rename it accordingly, `metastoreLoadBalancingEnabled`, also getters and setters